docs/system-management/agent/Ch_SysMgmtAgent.rst

```diff
@@ -5,4 +5,206 @@ System Management Agent (SMA)
 ============
 Introduction
 ============
-More documentation coming soon.
\ No newline at end of file
+
+While the SMA serves several purposes, it is to be considered, first and foremost, as the single connection point of management control for an EdgeX instance. As such, the API calls related to system management are defined so as to interface with the SMA.
+
+=====================
+Examples of API Calls
+=====================
+
+To get an appreciation for some SMA API calls in action, it will be instructive to look at what responses the SMA provides to the caller, for the respective calls (Notice, too, the error messages returned by the SMA, should it encounter a problem). Thus, consider the following calls that the SMA handles:
+
+* Metrics of a service
```

Comment:
If you make the section titles into headers (as requested below) we can also make this list into a set of links to each section.

See https://stackoverflow.com/questions/18632781/how-to-make-an-internal-link-to-a-heading-in-sphinx-restructuredtext-without-cre for how to do that

Reply:
- I've used the following `RST` syntax (and tested navigability) for creating linkages for sections, i.e. Implicit Links to Titles:
```
* `Metrics of a service`_
* `Configuration of a service`_
* `Start a service`_
* `Stop a service`_
* `Restart a service`_
* `Health check on a service`_
```